src/middlewared/middlewared/plugins/jail.py

```diff
@@ -631,6 +661,34 @@ def list_resource(self, resource, remote, want_cache, branch):
 
         return resource_list
 
+    @periodic(interval=86400)
+    @private
+    @accepts(Str('branch', null=True, default=None))
```

Comment:
`@private` and `@accepts` at the same time?

Reply:
Yup, we do that on other places as well, it's just to control the schema value being passed.